### PR TITLE
fixed most of package name

### PR DESCRIPTION
**Pull request by Nusk-Rbb**
I would appreciate a review of the code for the non-package fixes you mentioned in the issue.

### CODE REVIEW

File: arch.sh [hunk 1]
```diff
@@ -12,222 +17,75 @@ function install_metasploit() {
 function add_repos() {
     echo "Installing blackarch repo..."
     curl -O https://blackarch.org/strap.sh
-    bash strap.sh
+    if [[ `echo 26849980b35a42e6e192c6d9ed8c46f0d6d06047 strap.sh | sha1sum -c` == *"OK"* ]]; then
```
Comment (andreock) [inline arch.sh]: Can you remove this check? If strap.sh get updated this check will fail and the script stop working

**Comment by andreock** [comment]
I really like your refactor of the script, can you just remove the check of sha1 to avoid issue if blackarch update strap.sh?

**Comment by Nusk-Rbb** [comment]
Ok, Removed checksum strap.sh

**Review by andreock — APPROVED** [review]

**Comment by andreock** [comment]
Ok, thank you

**Pull request merged**